src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
```

Comment:
this is acceptable, but you can also consider making it private and providing DAO-like methods to put new value etc. it's a bit weird to make storage public and create getter for it